wit/deps/http/error.wit

```diff
@@ -0,0 +1,10 @@
+interface wasi-cloud-error {
```

Comment:
This file (and all the other keyvalue files) are already available under wit/deps/keyvalue.  Can you use those versions instead?

Reply:
Note that I updated the `wasi-keyvalue` branch of my Spin repo to match this new wit directory structure.

Reply:
I might have misunderstood but the host implementation of wasi-keyvalue is utilizing keyvalue wits that merged into wasi-http, thus why I copied wasi-keyvalues files to wasi-http.